### ISSUE TITLE
iOS: allow swiping left to right to go back

### ISSUE THREAD

**Issue by prabirshrestha** [binwiederhier/ntfy#312]
When viewing a single notification, swiping left to right should be equivalent to back button. This is common gesture for majority of the iOS apps. I found it cumbersome to click the back button at the top in  the header.

Mail and Safari on the phone are some examples this.

**Comment by binwiederhier** [comment]
I agree, good find. I think this has something to do with the custom navigation thingy I had to do. Thanks for reporting.

**Comment by jsaltzman** [comment]
This would be a very welcomed and helpful addition, I keep find myself swiping.

> I agree, good find. I think this has something to do with the custom navigation thingy I had to do. Thanks for reporting.

**Comment by binwiederhier** [comment]
This is merged and will be deployed in v1.8.x
